tests/functional/spec/summary-screen.spec.js

```diff
@@ -63,4 +63,29 @@ describe('RSI - summary screen edit test', function() {
     expect(rsiSummaryPage.getChangeInRetailTurnoverSummary()).to.contain('This is to test edit links on summary screen - edited')
 
   })
+  it('Given a 0 is entered in a currency field when on the summary page and the original page it still is shown as 0', function(done) {
```

Comment:
This test name isn't consistent with the others, should be something more like:
"Given the RSI survey 0102 when a 0 is entered in a currency field then the summary screen should show £0 and the original page should show 0"